articles/industry-4.0-and-cybersecurity/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/industry-4.0-and-cybersecurity/
+title: Industry 4.0 and Cybersecurity
+description: An introduction to Industry 4.0 through the enabling technologies, including fundamental principles of security, cybersecurity challenges, threats and insinuations.
+author: collins-agesa
+date: 2020-09-06T00:00:00-08:00
+topics: [Cybersecurity]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/industry-4.0-and-cybersecurity/hero.jpg
+    alt: bulma image example
+---
+Many technologies have emerged in recent times. These technologies include the Internet of Things, Big Data Analytics and Cyber-Physical Systems. They have enabled new and impactful initiatives such as Industry 4.0. this initiative is revolutionizing the manufacturing industry. Yet, the cybersecurity challenges associated with the technology have never been greater.
```

Comment:
The introduction is very generic and you introduce a lot of terms that are never defined. You also mention that Industry 4.0 revolutionizes manufacturing, yet don't specify how. I suggest giving an overview of industries that may be relevant, or rethinking your intro, and explaining why we should care about Industry 4.0 and cybersecurity in this section for context.